pymtl3/passes/backends/verilog/test/TranslationImport_stdlib_test.py

```diff
@@ -124,7 +124,6 @@ def tv_out( model, tv ):
 # Swap run_test_queue with modified version that calls the local do_test
 def _run_queue_test( do_test, test_func ):
 
-  assert False
```

Comment:
@ptpan Can you double check if the stdlib tests in /verilog uses the verilog backend for translation and yosys backend for yosys? The way it runs the test seems to be different now.

Reply:
Just updated that test case to work with the new queue testbench